Ros1 safety docks update (#171)
husarion/panther_ros: panther_manager/README.md

> **Warning**
> To allow your computer to be shutdown without the sudo password, ssh into it and execute
> (if needed replace **husarion** with username of your choice):
> ``` bash
> sudo su
> echo husarion 'ALL=(ALL) NOPASSWD: /sbin/poweroff, /sbin/reboot, /sbin/shutdown' | EDITOR='tee -a' visudo
> ```

#### Faults Handle

> After receiving a message on the `/battery` topic, the `panther_manager` node makes decisions about what needs to be done. For more information regarding power supply state refer to [adc_node](/panther_battery/README.md#battery-statuses) documentation.

Reviewer: ```suggestion
After receiving a message on the `/battery` topic, the `panther_manager` node makes decisions regarding safety measures. For more information regarding power supply state, please refer to [adc_node](/panther_battery/README.md#battery-statuses) documentation.
```

| Power Supply Health | Procedure                                                                                                                                                                                                          |
| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| GOOD                | -                                                                                                                                                                                                                  |
| UNKNOWN             | -                                                                                                                                                                                                                  |

> | OVERHEAT            | 1. Turn on the fan. <br> 2. If battery temperature is higher that 55.0 **&deg;C**, trigger an emergency stop and turn off AUX. <br> 3. If battery temperature is higher that 62.0 **&deg;C**, shut down the robot. |
> | DEAD                | Shut down the robot.                                                                                                                                                                                               |

Reviewer: ```suggestion
| OVERHEAT            | 1. Turn on the fan. <br> 2. If battery temperature is higher that 55.0 **&deg;C**, trigger an emergency stop and turn off AUX. <br> 3. If battery temperature is higher that 62.0 **&deg;C**, shutdown the robot. |
| DEAD                | Shutdown the robot.                                                                                                                                                                                               |
```
